rpc/lib/server/handlers_test.go

```diff
@@ -154,6 +154,80 @@ func TestRPCNotification(t *testing.T) {
 	require.Equal(t, len(blob), 0, "a notification SHOULD NOT be responded to by the server")
 }
 
+func TestRPCNotificationInBatch(t *testing.T) {
+	mux := testMux()
+	tests := []struct {
+		payload     string
+		expectCount int
+	}{
+		{
+			`[
+				{"jsonrpc": "2.0","id": ""},
```

Comment:
do we even need to test notification the second time? Shouldn't it be enough to test that we don't respond to it?

Reply:
I was just testing for different permutations of notifications in the messages - I could remove the second test case, but I wanted to ensure that single and multiple notifications are also ignored.

Reply:
I would have kept the test case. It doesn't harm and what you say makes sense to me. It might be worth to add a comment for the reasoning behind it. Le's not revert (again) it though :-)